stingray/modeling/gpmodeling.py

```diff
@@ -50,11 +650,22 @@ def get_kernel(kernel_type, kernel_params):
     kernel_type: string
```

Comment:
So, I think your code uses `kernel_type` to mean the power spectral shape, whereas the previous use was for the actual covariance function used in the GP, no? It'd be good to make sure these don't get confused?